SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Channel/Channel.cs

```diff
@@ -379,7 +379,28 @@ private Task SignalDeletingLogs()
                 }
                 try
                 {
-                    TriggerIngestion(state, logs, batchId);
+                    // Before sending logs, trigger the sending event for this channel
```

Comment:
Missing new line.

Reply:
No it's fine in .NET since `{` convention is different.